statsmodels/stats/weightstats.py

```diff
@@ -226,6 +226,79 @@ def std_mean(self):
         return std / np.sqrt(self.sum_weights - 1)
 
 
+    def quantile(self, probs):
+        """
+        Quantiles from a weighted sample.
+
+        Parameters
+        ----------
+        probs : array-like
+            A vector of probability points at which to calculate the
+            quantiles.  Each element of `probs` should fall in [0, 1].
+
+        Returns
+        -------
+        If the data are 1d, returns a pandas Series indexed by the
+        probability points, containing the quantiles.  If the data are
+        2d, returns a pandas DataFrame indexed by the probability
+        points, with each column containing the quantiles for the
+        corresponding column of the data.
+
```

Comment:
I think we need to mention that it uses simple averaging between points, i.e. midpoint.